<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wp4_5</class>
 <widget class="QWizardPage" name="Wp4_5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_93">
     <property name="text">
      <string>Отек</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QCheckBox" name="checkBox_89">
     <property name="text">
      <string>Боль</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_92">
     <property name="text">
      <string>Корки</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>Болезни губ (К13.)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_90">
     <property name="text">
      <string>Гиперемия</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBox_94">
     <property name="text">
      <string>Эрозия/Трещины</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="checkBox_95">
     <property name="text">
      <string>Язва</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="checkBox_96">
     <property name="text">
      <string>Некроз</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_91">
     <property name="text">
      <string>Гиперплазия</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>Симптомы болезни губ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QRadioButton" name="radioButton_22">
     <property name="text">
      <string>Нет симптомов болезни губ</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_88">
     <property name="text">
      <string>Требуются дополнительные исследования для выявления симптомов болезни губ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="checkBox_97">
     <property name="text">
      <string>Абсцесс/Флегмона</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
